src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -1233,6 +1506,243 @@ input ParkingFilter {
     select: [ParkingFilterOperation!]
 }
 
+"""
+Settings that control the behavior of itinerary filtering.  These are advanced settings and
+should not be set by a user through user preferences.
+"""
+input PlanItineraryFilterInput {
+    """
+    Itinerary filter debug profile used to control the behaviour of itinerary filters.
+    """
+    itineraryFilterDebugProfile: ItineraryFilterDebugProfile = OFF
+
+    """
+    Pick one itinerary from each group after putting itineraries that are `85%` similar together,
+    if the given value is `0.85`, for example. Itineraries are grouped together based on relative
+    the distance of transit travel that is identical between the itineraries (access, egress and
+    transfers are ignored). The value must be at least `0.5`.
+    """
+    groupSimilarityKeepOne: Ratio = 0.85
+
+    """
+    Pick three itineraries from each group after putting itineraries that are `68%` similar together,
+    if the given value is `0.68`, for example. Itineraries are grouped together based on relative
+    the distance of transit travel that is identical between the itineraries (access, egress and
+    transfers are ignored). The value must be at least `0.5`.
+    """
+    groupSimilarityKeepThree: Ratio = 0.68
+
+    """
+    Of the itineraries grouped to maximum of three itineraries, how much worse can the non-grouped
+    legs be compared to the lowest cost. `2.0` means that they can be double the cost, and any
+    itineraries having a higher cost will be filtered away. Use a value lower than `1.0` to turn the
+    grouping off.
+    """
+    groupedOtherThanSameLegsMaxCostMultiplier: Float = 2.0
```

Comment:
Do we actually want to expose them? What are they used for?

Reply:
There are two reasons for exposing these:
 - If you want to experiment with them.
 - These are probably the most important filter settings for transit. The end user should not be exposed, but a you might want to set these in a BFF. Note the doc on itinerary filter settings: 
 > Configure the itinerary-filter-chain. NOTE! THESE PARAMETERS ARE USED FOR SERVER-SIDE TUNING AND IS AVAILABLE HERE FOR TESTING ONLY.